src/params_serializer.js

```diff
@@ -30,15 +28,38 @@ const serialize = (key, value) => {
   } else if (value instanceof LatLngBounds) {
     v = `${value.ne.lat},${value.ne.lng},${value.sw.lat},${value.sw.lng}`;
   } else if (Array.isArray(value)) {
-    v = value;
+    v = _.map(value, serializeValue);
```

Comment:
Maybe use the built in map function for the array.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#browser_compatibility

```suggestion
    v = value.map(serializeValue);
```